src/modules/translate/translateApi.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { resolveJsonOrRejectWithError } from '../../util/apiHelpers';
+import config from '../../config';
+
+const baseUrl = `${
+  config.ndlaEnvironment === 'test'
+    ? 'https://cors-anywhere.herokuapp.com/'
```

Comment:
Hvorfor trengs denne proxyen bare i test?

Reply:
Det er for å unngå CORS error, men jeg tror det bare skjer med kall fra localhost.
https://stackoverflow.com/questions/10883211/deadly-cors-when-http-localhost-is-the-origin
https://medium.com/@dtkatz/3-ways-to-fix-the-cors-error-and-how-access-control-allow-origin-works-d97d55946d9

Reply:
Ah, da hadde det kanskje vært heldig å brukt `NODE_ENV` slik som vi gjør her https://github.com/NDLANO/editorial-frontend/blob/6ef9c0cc4ab221fab401b05e48dd51dc09f0c088/src/server/contentSecurityPolicy.js#L91

Da får vi testa CORS'en i testmiljøet når vi deployer, men unngår den når vi kjører dev-serveren.